contracts/TreasuryManager.sol

```diff
@@ -300,10 +301,14 @@ contract TreasuryManager is
         NOTE.transfer(address(0x000000000000000000000000000000000000dEaD), amountBought);
         emit NoteBurned(amountBought);
 
-        return amountSold;
+        return amountBought;
```

Comment:
Since you are not using the `amountSold` parameter and you are naming the `burnedNOTE` parameter, you can just destructure directly into `burnedNOTE` by doing the following:
```
(/* amountSold */, burnedNote) = 
            trade._executeTrade(uint16(DexId.ZERO_EX), TRADING_MODULE);
```

And you do not need the return statement.

Reply:
But I am using `amountSold` for TradeExecuted event